### PR TITLE
FEAT: Added Log-in Page

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-04-04 20-53-23

### CODE REVIEW

**Review by vinc3nati — APPROVED** [review]
good naming convention and component structure

File: src/pages/SignIn/SignIn.jsx [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
-
+import { Link } from 'react-router-dom'
+import {NavBar,Footer} from '../HomePage/HomePageComponents'
+import '../SignIn/SignIn.css'
 function SignIn() {
 return (
 <div>
-    <h1>Blank SignIn page for Route checking</h1>
+    <NavBar />
+    {/* <h1>Blank SignIn page for Route checking</h1> */}
```
Comment (ayush65) [inline src/pages/SignIn/SignIn.jsx:9]: remove commented code

**Comment by Adarsh-Patel-Dev** [comment]
classnames are consistent and overall code is fine

File: src/pages/SignIn/SignIn.jsx [hunk 2]
```diff
@@ -1,10 +1,39 @@
...
+    <div className="login-container">
+        <h1 className="header-large">LOG IN</h1>
+        <div className="input">
```
Comment (MaruthiPothuganti) [inline src/pages/SignIn/SignIn.jsx:12]: Have a unique classNames. This might mislead.

**Review by ayush65 — APPROVED** [review]
code looks clean 
Class Names are self Explanatory

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Good structuring of components . Looks good to merge.

File: src/pages/SignIn/SignIn.jsx (hunk 1 above)
Comment (kotesh-arya) [reply src/pages/SignIn/SignIn.jsx:9]: sure 🙂

File: src/pages/SignIn/SignIn.jsx (hunk 2 above)
Comment (kotesh-arya) [reply src/pages/SignIn/SignIn.jsx:12]: Sure, Will update them 👍️

File: src/pages/SignIn/SignIn.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    text-align: left;
+    width: 25%;
+    text-align: center;
+    box-shadow: 2px 2px 10px var(--primary-color);
+    margin: auto;
+    margin-top: 10rem;
+    padding: 3rem;
+    margin-bottom: 4.1rem;
+    font-size: var(--large-font-size);
+}
```
Comment (Rockyraven) [inline src/pages/SignIn/SignIn.css:11]: use margin consitancy 
it will reduce your code

File: src/pages/SignIn/SignIn.jsx [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+            <input className="input-box basic" type="password" placeholder="************"/>
+        </div>
+        <div className="handy-div">
+            <div className="checkbox-text">
+                <input className="check-box" type="checkbox"/>
+                <label className="check-label" htmlFor="check-box">Remember Me</label>
+            </div>
+            {/* <a  href="">Forgot Password</a> */}
+            <Link className="forgot-password" to="">Forgot Password</Link>
+        </div>
```
Comment (Rockyraven) [inline src/pages/SignIn/SignIn.jsx:26]: remove that unwanted code

**Review by Rockyraven — COMMENTED** [review]
Do some minar change
otherwise your code is clean and easy to readable
you are good to go

**Pull request merged**